### PR TITLE
Fixes to the CI document

### LINKED ISSUE #1454: Clean up the CI document

### PR DESCRIPTION
**Pull request by guardrex**
Various fixes to the CI document.

Replaces #1556 
Fixes #1454

### CODE REVIEW

**Comment by GuardRex** [comment]
Stand-by ... I'll see if I can get that failure fixed.

**Comment by GuardRex** [comment]
@mairaw Stand by on this one for a sec. I'm doing a Rex Tune-up:tm: on it. One more pass on it when I get back from the :muscle:, and it will be ready for a look.

**Comment by GuardRex** [comment]
@mairaw I'll probably give it one more peek in the morning, but it's good enough now for a 1st draft look :eye::eye:. The PS script works, but I haven't tried the bash script. I should be able to give that a quick test here, so I'll do that in the morning just to make sure it doesn't :boom:.

**Comment by GuardRex** [comment]
@mairaw Yes, the bash script required a touch of work, too. There's a question there tho on a problem at the end of the script getting the path to `dotnet` stored into a var for later use. Who is our go-to for bash-related questions?

Except for the little bash problem, this is ready for :eyes:.

**Comment by GuardRex** [comment]
@mairaw @rpetrusha Nice ... a little extra polish on this last pass. It's ready for :eyes: now.

File: docs/core/tools/using-ci-with-cli.md [hunk 1]
```diff
@@ -15,93 +15,169 @@ ms.assetid: 0d6e1e34-277c-4aaf-9880-3ebf81023857
...
+Write-Host "Downloading and installation of the SDK is complete."
+
+# $LocalDotnet holds the path to dotnet.exe for future use by the 
+#   script.
+$LocalDotnet = "$InstallDir/dotnet"
+
+# Run the build process now. Implement your build script here.
+```
+
+You provide the implementation for your build process at the end of the script. When run and after the tooling is acquired, the script executes your build process. For UNIX machines, the following bash script performs the actions described in the PowerShell script in a similar manner:
```
Comment (BillWagner) [inline docs/core/tools/using-ci-with-cli.md]: nit: "When run and after tooling is acquired" seems awkward. Maybe say "The script acquires the tools and then executes your build process."

**Review by BillWagner — APPROVED** [review]
@GuardRex This looks great.

I had one wording suggestion. See what you think. Then, :shipit:

**Pull request merged**